### ISSUE TITLE
diff between two exact same DateTimes

### ISSUE THREAD

**Issue by BaggioWongHK** [moment/luxon#329]
![image](https://user-images.githubusercontent.com/28295117/46028902-ca4b0200-c0bf-11e8-85da-5dd300361094.png)

As you can see if we have 

```
const dateA = luxon.DateTime.fromISO('2000-01-01T00:00');
const dateB = luxon.DateTime.fromISO('2000-01-01T00:00');
dateA.diff(dateB, ['hours']).toObject(); // <-- this will be an empty object
```

My question is what is the reasoning behind this decision for returning an empty object instead of `{ hours: 0 }`?

Also, perhaps a suggestion to add this to the documentation? As far as I can see, I don't see this documented in the docs. 

Thank you. :)

**Comment by icambron** [comment]
Hmm, yeah, I think `{ hours: 0 }` would be better. I'm not sure why it's like that.

**Comment by BaggioWongHK** [comment]
Ah I see, I thought this was a deliberate design decision, so I was curious. Thanks for getting back to me. 😃

**Comment by icambron** [comment]
I'd rather just fix it :)

**Comment by icambron** [comment]
Fixed in cfa3f99670f9bd9fdd184e77cdded9b391d34c95, released in 1.4.1
